Proposal by Serena Caplins

TITLE

De novo genome assembly for a reef building polychaete worm

Abstract

The dispersal of propagules from their natal site to a settlement site has lasting implications not only for individual fitness, but also population fitness and persistence. Dispersal is of particular interest in marine communities, where sessile adults release thousands of larvae to the water column. How far a larvae will travel and whether closely related individuals (e.g., siblings, half-siblings) recruit to the same site, are long-standing questions in the biology of dispersal. The reef building annelid worm Phragmatopoma californica (Polychaete, Sabellarididae) is a common inhabitant along the intertidal zone of the rocky coast of California. P. californica dwell in sand tubes held together by a secreted glue-like substance and are typically found in aggregates that can form large reefs, drastically changing the form of the environments in which they occur. While direct observations of larval dispersal patterns are impractical if not impossible, assessing genetic connectedness and relatedness of individuals across multiple spatial scales will enable the inference of these patterns. Specifically, I will examine if relatedness decreases across increasing distances. Or alternatively, are aggregations panmictic assemblages of individuals? These questions will be addressed with microsatellite markers developed from paired-end illumina sequence data. 
I am requesting access to the computing power of XSEDE to implement a de novo assembly using existing un-assembled paired-end illumina data consisting of approximately 2 million reads and an estimated 5x coverage. Assembly will be performed with the Velvet algorithm package. 
SSRs will be identified with msatcommander and will be screened for the number and type of repeat. Repeats greater in length than the insert size or with a very common occurrence will be rejected for future use due to the potential unreliability of their correct assembly. 
I also request access to XSEDE to perform some exploratory procedures such as mapping previously sequenced genes (e.g., cement precursor protein mRNA, ITS1 rDNA, CO1 mtDNA) to the de novo assembly generated herein, with the aim of identifying and characterizing unknown regions that flank known sequences. Mapping will be performed with Burrows-Wheeler Aligner (BWA). 

Keywords: 

de novo assembly, illumina, paired-end, SSR, relatedness, connectivity, 
